### PR TITLE
Adding CloudRoast configuration

### PR DESCRIPTION
**Pull request by xeregin**

### CODE REVIEW

File: rpcd/playbooks/roles/cloudroast/CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Extra
+-----
+
+Tags:
+    If it's a bug that needs fixing in a branch in addition to Master, add a '\<release\>-backport-potential' tag (eg ``juno-backport-potential``).  There are predefined tags that will autocomplete.
+
+Status:
+    Please leave this alone, it should be New till someone triages the issue.
+
+Importance:
```
Comment (nrb) [inline rpcd/playbooks/roles/cloudroast/CONTRIBUTING.rst]: I don't think this part is relevant to things hosted on Github.

**Comment by sigmavirus24** [comment]
This *looks* okay. I haven't tried it out yet though.

**Comment by hughsaunders** [comment]
looks like scripts/maas_setup_configuration.py is failing on flake8

**Comment by sigmavirus24** [comment]
Triggering new run of Travis with new linting/Travis config

**Comment by sigmavirus24** [comment]
https://travis-ci.org/rcbops/rpc-openstack/jobs/95368249 should tell you exactly what needs fixing

**Pull request closed without merging**